apps/backend/test/elections/get_election.test.ts

```diff
@@ -0,0 +1,89 @@
+import {
+  response404Object,
+  response406Object,
+  type SelectableElection,
+  selectableElectionObject,
+  type SelectableUser,
+} from '@repo/votura-validators';
+import request from 'supertest';
+import { beforeAll, describe, expect, it } from 'vitest';
+import { app } from '../../src/app.js';
+import { HttpStatusCode } from '../../src/httpStatusCode.js';
+import { createElection } from '../../src/services/elections.service.js';
+import { createUser, findUserBy } from '../../src/services/users.service.js';
+
+describe('GET /elections/:electionId', () => {
+  const AUTH_TOKEN = '1234';
+  const ELECTIONS_SLUG = '/elections';
+
+  let user: SelectableUser | null = null;
+  let election: SelectableElection | null = null;
+
+  beforeAll(async () => {
+    await createUser({
+      email: 'user@votura.org',
+      password: 'password',
+    });
+
+    user = await findUserBy({
+      email: 'user@votura.org',
+    });
+
+    if (user === null) {
+      throw new Error('User not found!');
+    }
+
+    election = await createElection(
+      {
+        private: true,
+        name: 'Election 1',
+        description: 'This is election one',
+        votingStartAt: '2024-07-29T15:51:28.071Z',
+        votingEndAt: '2024-07-30T15:51:28.071Z',
+        allowInvalidVotes: false,
+      },
+      user?.id,
```

Comment:
```suggestion
      user.id,
```
You checked already for `null`.

Same for `election` below.

Reply:
For the user the `?` was not necessary, but the code below does not know that we are assigning non `null` values to these variables.